eras/babbage/impl/src/Cardano/Ledger/Babbage/TxBody/Internal.hs

```diff
@@ -186,22 +186,38 @@ instance
   EqRaw (BabbageTxBodyRaw era)
   where
   eqRaw a b =
-    btbrInputs a == btbrInputs b
-      && btbrCollateralInputs a == btbrCollateralInputs b
-      && btbrReferenceInputs a == btbrReferenceInputs b
-      && btbrOutputs a `eqSeqUnsized` btbrOutputs b
-      && btbrCollateralReturn a `eqMbUnsized` btbrCollateralReturn b
-      && btbrTotalCollateral a == btbrTotalCollateral b
-      && btbrCerts a == btbrCerts b
-      && btbrWithdrawals a == btbrWithdrawals b
-      && btbrFee a == btbrFee b
-      && btbrValidityInterval a == btbrValidityInterval b
-      && btbrUpdate a == btbrUpdate b
-      && btbrReqSignerHashes a == btbrReqSignerHashes b
-      && btbrMint a == btbrMint b
-      && btbrScriptIntegrityHash a == btbrScriptIntegrityHash b
-      && btbrAuxDataHash a == btbrAuxDataHash b
-      && btbrNetworkId a == btbrNetworkId b
+    btbrInputs a
+      == btbrInputs b
+      && btbrCollateralInputs a
```

Comment:
This reformatting is indicator of older version of foumolu applied to the module first